src/mono/wasm/timezones.props

```diff
@@ -0,0 +1,12 @@
+<Project>
+  <ItemGroup Condition="'$(TargetOS)' == 'Browser'">
+    <_TimeZonesToBundle Include="africa" />
+    <_TimeZonesToBundle Include="antarctica" />
+    <_TimeZonesToBundle Include="asia" />
+    <_TimeZonesToBundle Include="australasia" />
+    <_TimeZonesToBundle Include="etcetera" />
+    <_TimeZonesToBundle Include="europe" />
+    <_TimeZonesToBundle Include="northamerica" />
+    <_TimeZonesToBundle Include="southamerica" />
```

Comment:
nit: IMO as this file is now only imported by callers that need it, I would rename them item to `TimeZoneToBundle` (without the underscore prefix and also without the plural as msbuild items are singular by default).